src/cargo/core/compiler/mod.rs

```diff
@@ -614,7 +614,7 @@ fn rustdoc(cx: &mut Context<'_, '_>, unit: &Unit) -> CargoResult<Work> {
                 &mut |line| on_stderr_line(state, line, package_id, &target, &mut output_options),
                 false,
             )
-            .chain_err(|| format!("Could not document `{}`.", name))?;
+            .chain_err(|| format!("could not document `{}`.", name))?;
```

Comment:
```suggestion
            .chain_err(|| format!("could not document `{}`", name))?;
```

and maybe also remove the `.` from the `could not compile` message that appears above in the same file?  That change will require updating a few tests.